data/borderlines.json

```diff
@@ -197,6 +212,21 @@
           "name": "Everest Fun Park & Resort Pvt. Ltd.",
           "latitude": 26.6431722,
           "longitude": 87.6181256
+        },
+        {
+          "name": "Urlabari Multiple Campus",
+          "longitude": 26.6651,
+          "latitude": 87.6200
+        },
+        {
+          "name": "Buddhasubba Temple",
+          "longitude": 26.82193,
+          "latitude": 87.29652
+        },
+        {
+          "name": "Kosi Barrage",
```

Comment:
Koshi Barrage isn't in urlabari

Reply:
Hi @thenishantsapkota 
Sorry, I relied on specific websites earlier. I've modified it with Urlabari Chowk from Google Maps